ui/qt-lib/src/Ariadne/UI/Qt/Widgets/Settings.hs

```diff
@@ -0,0 +1,336 @@
+  connect_ aboutButton QAbstractButton.clickedSignal $ \_ -> showWidget st aboutWidget aboutButton
+  connect_ supportButton QAbstractButton.clickedSignal $ \_ -> showWidget st supportWidget supportButton
+  return (settings, st)
+
+initGeneralSettings :: IO (QWidget.QWidget, GeneralSettings)
+initGeneralSettings = do
+  generalWidget <- QWidget.new
+  generalLayout <- QVBoxLayout.new
+  QLayout.setContentsMarginsRaw generalLayout 42 42 24 42
+
+
+  languageLabel <- QLabel.newWithText ("<b>LANGUAGE</b>" :: String)
+  languageSelector <- QComboBox.new
+  QComboBox.addItem languageSelector ("English" :: String)
+  addRow generalLayout languageLabel languageSelector
+  addSeparator generalLayout
+  
+
+  countervalueLabel <- QLabel.newWithText ("<b>COUTNERVALUE</b>" :: String)
+  countervalueSelector <- QComboBox.new
+  QComboBox.addItem countervalueSelector ("USD" :: String)
+  addRow generalLayout countervalueLabel countervalueSelector
+  addSeparator generalLayout
+
+  rateProviderLabel <- QLabel.newWithText ("<b>RATE PROVIDER</b>" :: String)
+  rateProviderSelector <- QComboBox.new
+  QComboBox.addItem rateProviderSelector ("Bittrex" :: String)
+  addRow generalLayout rateProviderLabel rateProviderSelector
+  addSeparator generalLayout
+
+  themeLabel <- QLabel.newWithText ("<b>THEME</b>" :: String)
+  themeSelector <- QComboBox.new
+  QComboBox.addItem themeSelector ("Mint" :: String)
+  addRow generalLayout themeLabel themeSelector
+
+  for_ [languageSelector, countervalueSelector, rateProviderSelector, themeSelector] $ \w ->
+    QWidget.setSizePolicyRaw w Maximum Fixed
+
+  for_ [languageSelector, countervalueSelector, rateProviderSelector, themeSelector] $ \w ->
+    QWidget.setEnabled w False
+
+  QWidget.setLayout generalWidget generalLayout
+
+
+  let gs = GeneralSettings{..}
+
+  return (generalWidget, gs)
+
+
+initAboutSettings :: IO (QWidget.QWidget, AboutSettings)
+initAboutSettings = do
+  aboutWidget <- QWidget.new
+  aboutLayout <- QVBoxLayout.new
+  
+  let currentVersion = "Ariadne 1.0.2"
+  
+  versionLabel <- createTwoLinesLabel "VERSION" currentVersion
+  releaseNotesButton <- QPushButton.newWithText("Release notes" :: String)
+  void $ setProperty releaseNotesButton ("styleRole" :: Text) ("rightPaneSettingsButton" :: Text)
+  addRow aboutLayout versionLabel releaseNotesButton
+  addSeparator aboutLayout
+
+
+  let license = "Sample text about license"
+
+  licenseLabel <- createTwoLinesLabel "LICENSE" license
+  licenseButton <- QPushButton.new
+  void $ setProperty licenseButton ("styleRole" :: Text) ("linkButton" :: Text)
+  addRow aboutLayout licenseLabel licenseButton
+
+  for_ [releaseNotesButton, licenseButton] $ \w ->
+    QWidget.setSizePolicyRaw w Maximum Fixed
+
+  QWidget.setLayout aboutWidget aboutLayout
+
+  let as = AboutSettings{..}
+
+  connect_ releaseNotesButton QAbstractButton.clickedSignal 
+        $ \_ -> (openBrowser "https://github.com/serokell/ariadne/blob/master/CHANGELOG.md" >> pass)
```

Comment:
Ideally, this shouldn't be hardcoded, but passed from backend, like update url is passed. @gromakovsky ?